ReactWindows/ReactNative/Views/Image/ReactImageManager.cs

```diff
@@ -22,6 +24,9 @@ public class ReactImageManager : SimpleViewManager<Border>
 
         private readonly IImageCache _imageCache;
 
+        private readonly Dictionary<int, Dictionary<string, double>> _imageSources =
```

Comment:
>int [](start = 36, length = 3)

If you're going to key on the tag for this one, you might as well key on the tag for _disposables.